cmd/broker/main.go

```diff
@@ -23,12 +24,20 @@ func main() {
 
 	configFilepath := os.Args[1]
 	servicePath := os.Args[2]
+
 	state, err := config.NewState(configFilepath, servicePath)
 	if err != nil {
 		logger.Fatal("main", err)
 	}
 
-	broker := broker.NewBlockheadBroker(state)
+	cli, err := client.NewEnvClient()
+	if err != nil {
+		logger.Fatal("docker-client", err)
+	}
+
+	manager := broker.NewDockerContainerManager(logger, cli)
+
+	broker := broker.NewBlockheadBroker(logger, state, manager)
```

Comment:
since we have an object with a logger specific to us, we can use anything we want, and let the other logger do what it wants.